packages/web-components/src/button/button.styles.ts

```diff
@@ -308,4 +308,11 @@ export const styles = css`
     color: ${colorNeutralForegroundDisabled};
     cursor: not-allowed;
   }
-`;
+`.withBehaviors(
+  forcedColorsStylesheetBehavior(css`
+    :host(:hover) .control,
+    :host([appearance='transparent']:hover) .control {
```

Comment:
@brianchristopherbrady what is the reason we need both of these? `:host(:hover) .control` also applies to transparent, does it not?

Reply:
`:host(:hover) .control` this was not applying the style to transparent. But after a 2nd look it seems it wasn't applying styles to any of the variations and isn't needed. But  the `:host([appearance='transparent']:hover) .control` selector is still necessary for the transparent variation.